x-pack/plugins/spaces/server/default_space/create_default_space.ts

```diff
@@ -48,6 +50,7 @@ export async function createDefaultSpace({ getSavedObjects, logger }: Deps) {
         color: '#00bfb3',
         disabledFeatures: [],
         _reserved: true,
+        ...(solution ? { solution } : {}),
```

Comment:
nit: `solution` would be enough here I think

```ts
disabledFeatures: [],
_reserved: true,
solution,
...
```

Reply:
we have [dynamic schema](https://github.com/elastic/kibana/pull/183986/files#diff-f62fbc294f4205143f9e63aaae5363d4a63b3f1c15de6a39edee010eeb2d9f5aR54) validation and for serverless specifying solution key is forbidden, so it is intended here. 
Through probably we can convert it to serverless check specifically

Reply:
@sebelga since you already looked at the PR, may I ask a question? 🙂 
I wonder if just [that line](https://github.com/elastic/kibana/pull/185926/files#diff-4983d418b5d571bf1af913a4fdd4a7d78117d6141afefb7499d852391832050cR166) would be sufficient for requirements.
The default solution should be defined only for ESS, but if I get it correct that setting would be undefined in serverless and there is probably no way to define it through cloud plugin on prem?